app/models/validations/date_validations.rb

```diff
@@ -53,9 +53,6 @@ def validate_startdate(record)
     if record["mrcdate"].present? && record["startdate"].to_date - record["mrcdate"].to_date > 3650
       record.errors.add :startdate, I18n.t("validations.setup.startdate.ten_years_after_mrc_date")
     end
-
-    location_during_startdate_validation(record)
-    scheme_during_startdate_validation(record)
```

Comment:
on this note, do we need `location_during_startdate_validation` to be duplicated in `validate_scheme` in setup_validations.rb?

Reply:
Nope, removed it too!